src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -60,18 +64,21 @@ export interface IEditorPageState {
     selectedTag: string;
     /** Tags locked for region labeling */
     lockedTags: string[];
+    thumbnailSize: ISize;
```

Comment:
missing comment

Nit: we should refactor existing comment into proper jsdoc

Reply:
These are proper JS doc comments for the properties and the Storybook JS doc plugin expect it in this format.  Added missing doc.